services/ui-src/src/forms/mcpar/dpc/dpc.json

```diff
@@ -32,6 +26,24 @@
           "hint": "What is the total number of individuals enrolled in each plan as of the first day of the last month of the reporting year?",
           "mask": "comma-separated"
         }
+      },
+      {
+        "id": "dpc-2",
+        "type": "number",
+        "props": {
+          "label": "D1.I.2 Plan share of Medicaid",
+          "hint": "<p>Automatically calculated. Sum of plan enrollment (within the specific program) as a percentage of the state's total Medicaid enrollment.</p><ul><li>Numerator: Plan enrollment (D1.I.1)</li><li>Denominator: Statewide Medicaid enrollment (B.I.1)</li><ul><li>System will need to calculate after entry in D1.I.1 (non editable)</li></ul></ul>",
```

Comment:
The nested `<ul>` is not necessary -- just the numerator and denominator. Same comment for `hint` below.